Lexplorer/Components/HomepageOverview.razor

```diff
@@ -85,8 +85,8 @@
 </MudGrid>
 
 <MudGrid>
-    <MudItem sm="5">
-        <MudTable Dense="true" Items="@blockData?.data?.blocks" Hover="true">
+    <MudItem xs="12" sm="12" md="4">
+        <MudTable Items="@blockData?.data?.blocks" Hover="true">
```

Comment:
Why did you remove the Dense for the blocks table, but not for the transactions and the pairs?

Reply:
That's a glitch by my side. Set it to be Dense again in latest commit.